tracer/build/_build/UpdateVendors/VendoredDependency.cs

```diff
@@ -123,6 +123,13 @@ static VendoredDependency()
                 downloadUrl: "https://github.com/DataDog/dotnet-vendored-code/archive/refs/tags/1.0.0.zip",
                 pathToSrc: new[] { "dotnet-vendored-code-1.0.0", "System.Reflection.Metadata", "System.Runtime.CompilerServices.Unsafe" },
                 transform: filePath => RewriteCsFileWithStandardTransform(filePath, originalNamespace: "System.Runtime", AddNullableDirectiveTransform, AddIgnoreNullabilityWarningDisablePragma));
+
+            Add(
+                libraryName: "ICSharpCode.SharpZipLib",
+                version: "1.3.3",
+                downloadUrl: "https://github.com/icsharpcode/SharpZipLib/archive/refs/tags/v1.3.3.zip",
+                pathToSrc: new[] { "SharpZipLib-1.3.3", "src", "ICSharpCode.SharpZipLib" },
+                transform: filePath => RewriteCsFileWithStandardTransform(filePath, originalNamespace: "ICSharpCode.SharpZipLib"));
```

Comment:
can/should we vendor this in _only_ for .NET FX 🤔 We only need it there (as I understand it) and it would reduce bloat in the dll size?

Reply:
Yep! I was able to isolate the vendoring changes so that it only affects .NET FX builds in 59971aa153eadac7fd3f9dcf0cfc51b3a79da14c